lab_5_scrapper/scrapper.py

```diff
@@ -120,45 +231,85 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
+        self.article = Article(self.full_url, self.article_id)
 
     def _fill_article_with_text(self, article_soup: BeautifulSoup) -> None:
         """
         Finds text of article
         """
-        pass
+        text = article_soup.find('span', {'itemprop': 'articleBody'}).find_all('p')
+        self.article.text = '\n'.join([p.get_text(strip=True) for p in text])
 
     def _fill_article_with_meta_information(self, article_soup: BeautifulSoup) -> None:
         """
         Finds meta information of article
         """
-        pass
+        title = article_soup.find('h1')
+        if title:
```

Comment:
title must always be filled
as of now, it looks like there might be cases where title field will not be filled, it is unacceptable